modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/sidecar/ProcessExecutorPathsImpl.java

```diff
@@ -32,7 +32,8 @@ public ProcessExecutorPathsImpl(Props props) {
 	@Override
 	public Path getLibPath() {
 		return Paths.get(
-			_props.get(PropsKeys.LIFERAY_SHIELDED_CONTAINER_LIB_PORTAL_DIR));
+			_props.get(
```

Comment:
@W2jx Based on the change in your current PR, you should let this use `ModuleFrameworkPropsValues.LIFERAY_SHIELDED_CONTAINER_LIB_PORTAL_DIR` directly.